src/vos/vos_obj_cache.c

```diff
@@ -160,18 +160,19 @@ vos_obj_cache_create(int32_t cache_size, struct daos_lru_cache **occ)
 {
 	int	rc;
 
-	D_DEBUG(DB_TRACE, "Creating an object cache %d\n", (1 << cache_size));
 	rc = daos_lru_cache_create(cache_size, D_HASH_FT_NOLOCK,
 				   &obj_lru_ops, occ);
-	if (rc)
-		D_ERROR("Error in creating lru cache: "DF_RC"\n", DP_RC(rc));
+	DL_CDEBUG(rc != 0, DLOG_ERR, DLOG_INFO, rc, "Create VOS object LRU cache %p, size %u",
+		  *occ, 1 << cache_size);
+
 	return rc;
 }
 
 void
 vos_obj_cache_destroy(struct daos_lru_cache *occ)
 {
 	D_ASSERT(occ != NULL);
+	D_INFO("Destroy VOS object LRU cache %p\n", occ);
```

Comment:
Same here, this INFO message should be removed or changed to DB_TRACE. We really shouldn't print an internal hash create/destroy message in INFO level.

Reply:
> Same here, this INFO message should be removed or changed to DB_TRACE. We really shouldn't print an internal hash create/destroy message in INFO level.

DB_TRACE is inconvenient for user since they usually will not enable trace level debug.